main.go

```diff
@@ -82,5 +79,9 @@ func main() {
 			}
 			fmt.Printf("%s\n", bytes.Join(output, delimiter))
 		}
+
+		if len(content) < 1 {
```

Comment:
Please explain why this should be `if readAllErr != io.EOF {`.

Reply:
shouldnt it be if readAllErr == io.EOF { ?

Reply:
You're right!